server/src/main/java/io/spine/server/commandbus/CommandBus.java

```diff
@@ -237,10 +205,40 @@ private static IllegalStateException noDispatcherFound(CommandEnvelope commandEn
     }
 
     /**
+     * {@inheritDoc}
+     *
+     * <p>Performs the checks defined by the {@code filterChain}. If the message passes the filters,
+     * it is been posted into the bus and stored into the associated {@link CommandStore}.
+     */
+    @Override
+    protected boolean prepareAndPost(Command command, StreamObserver<Response> responseObserver) {
+        final CommandEnvelope commandEnvelope = CommandEnvelope.of(command);
+        if (!filterChain.accept(commandEnvelope, responseObserver)) {
+            return false;
+        }
+        commandStore().store(command);
+        responseObserver.onNext(Responses.ok());
+
+        doPost(commandEnvelope);
+        return true;
+    }
+
+    /**
+     * {@inheritDoc}
+     *
+     * <p>Performs no action.
+     */
+    @Override
+    protected void store(Command command) {
+        // NoOp
```

Comment:
Please expand this comment. And I don't mean `// No operation.` :)  

Instead, tell readers why. It's really confusing, given that a few lines above you have `commandStore().store(command)`.